integration_tests/upgrade/Dockerfile-build

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 DSR Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+############################
+# build upgraded app
+############################
+FROM dcld-build
+
+WORKDIR /go/src/dc-ledger
+
+RUN echo -e \
+'package keeper\n'\
```

Comment:
Probably makes sense to put that code to separate .go files and just copy it?

Reply:
This would require to keep such files up-to-date on every update in `app/app.go` and also in `x/model/module.go`.

The dynamic changes being applied in this Dockerfile are more flexible and further can be improved to not require any maintenance at all (after the first real migration in `model` module is introduced).